docs/middleware/logger.md

```diff
@@ -88,6 +88,27 @@ app.Use(logger.New(logger.Config{
 app.Use(logger.New(logger.Config{
     DisableColors: true,
 }))
+
+// Use Custom Logger with Fiber Logger Interface
+func LoggerToWriter(customLogger fiberlog.AllLogger, level fiberlog.Level) io.Writer {
+	return &struct {
+		Write func(p []byte) (n int, err error)
+	}{
+		Write: func(p []byte) (n int, err error) {
+			switch level {
+			case fiberlog.LevelDebug:
+				customLogger.Debug(string(p))
+			case fiberlog.LevelError:
+				customLogger.Error(string(p))
```

Comment:
Can you add support for other logging levels too?

Reply:
@coderabbitai pls prepare this change